proxywasm/v2/imports.go

```diff
@@ -59,6 +63,7 @@ func RegisterImports(instance common.WasmInstance) {
 	_ = instance.RegisterFunc("env", "proxy_increment_metric_value", ProxyIncrementMetricValue)
 	_ = instance.RegisterFunc("env", "proxy_delete_metric", ProxyDeleteMetric)
 
+	_ = instance.RegisterFunc("env", "proxy_http_call", ProxyDispatchHttpCall)
 	_ = instance.RegisterFunc("env", "proxy_dispatch_http_call", ProxyDispatchHttpCall)
```

Comment:
delete "proxy_dispatch_http_call"?

Reply:
in v1/imports.go, [codes](https://github.com/mosn/proxy-wasm-go-host/blob/main/proxywasm/v1/imports.go#L61) like 
```GO
	_ = instance.RegisterFunc("env", "proxy_http_call", ProxyHttpCall)
	_ = instance.RegisterFunc("env", "proxy_dispatch_http_call", ProxyHttpCall)
```
Maybe we can keep it？

Reply:
ok